pkg/apis/servicecatalog/v1alpha1/types.go

```diff
@@ -328,6 +340,14 @@ type InstanceSpec struct {
 	//
 	// Immutable.
 	ExternalID string `json:"externalID"`
+
+	// Currently, this field is ALPHA: it may change or disappear at any time
+	// and its data will not be migrated.
+	//
+	// AlphaUser contains information about the user that created this
+	// instance. This field is set by the API server and not settable by the
+	// end-user. User-provided values for this field are not saved.
+	AlphaUser AlphaUserInfo `json:"alphaUserInfo,omitempty"`
```

Comment:
If you want to position field as alpha, and require a feature gate to enable it in the server, that can work. Also, if the thing you are capturing is the creating user's info, I'd name the field to reflect that